src/backend/vulkan/src/lib.rs

```diff
@@ -143,7 +143,7 @@ impl Instance {
             application_version: version,
             p_engine_name: b"gfx-rs\0".as_ptr() as *const _,
             engine_version: 1,
-            api_version: 0, //TODO
+            api_version: 0, //TODO: VK_API_VERSION_1_1
```

Comment:
heh

Reply:
Actually, 0 is fine

Reply:
it spams a warning to console now (unrelated to the PR): 
> Warning: Unrecognized CreateInstance->pCreateInfo->pApplicationInfo.apiVersion number -- (0x00000000) assuming VK_API_VERSION_1_1.

Reply:
Sounds like a validation layer bug